src/common/types.ts

```diff
@@ -28,6 +28,7 @@ export interface IPlugin {
   name: string;
   description: string;
   usage: string;
+  pluginCommands: string[];
```

Comment:
Why not make this optional so that plugins aren't forced to have an empty array?

Reply:
Also call it `alias` not commands.

Reply:
When I made the interface it was the first thing I did, and at the time I didn't know how to go around the if null process for a call without it been redundant.  However since I add the main command also to the list I can make it unrequired, just that I didn't consciously realize that I had solved the initial problem.  I will add it in an update.